src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/InvokeRestMethodCommand.Common.cs

```diff
@@ -89,22 +89,17 @@ internal override void ProcessResponse(HttpResponseMessage response)
                 }
                 else
                 {
-                    // determine the response type
+                    // Determine the response type
                     RestReturnType returnType = CheckReturnType(response);
 
                     // Try to get the response encoding from the ContentType header.
-                    Encoding encoding = null;
-                    string charSet = response.Content.Headers.ContentType?.CharSet;
-                    if (!string.IsNullOrEmpty(charSet))
```

Comment:
You are not doing this check anymore. In case `charSet` is null, you will get an argument exception from `Encoding.GetEncoding`.

Reply:
It works correctly, test:
```ps1
$x = Invoke-WebRequest https://httpbin.org/get
$x.BaseResponse.Content.Headers.ContentType.MediaType
#--> application/json
$x.BaseResponse.Content.Headers.ContentType.CharSet
#--> No CharSet
Invoke-RestMethod https://httpbin.org/get
#--> json, no errors
```
Description:
- CharSet = **NULL**
- str = StreamHelper.DecodeStream(responseStream, **NULL**, out Encoding encoding);
- TryGetEncoding(**NULL**, out encoding)
- the exception is caught and encoding is set to null; TryGetEncoding returns false
- encoding is set to ContentHelper.GetDefaultEncoding() [UTF8]

I think this is the desired behaviour

Reply:
Good point. Thanks!